src/components/BulkEnrollmentPage/index.jsx

```diff
@@ -1,15 +1,43 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import Hero from '../Hero';
+import { Container } from '@edx/paragon';
 
+import { Switch, Route } from 'react-router-dom';
+import Hero from '../Hero';
+import { MultipleSubscriptionsPage, SubscriptionData } from '../subscriptions';
 import CourseSearch from './CourseSearch';
 
 function BulkEnrollmentPage({ enterpriseId }) {
   return (
     <div className="container-fluid bulk-enrollment">
       <Hero title="Catalog Management" />
-      <CourseSearch enterpriseId={enterpriseId} />
+      <SubscriptionData enterpriseId={enterpriseId}>
+        <main role="main" className="manage-subscription">
+          <Switch>
+            <Route
+              path="/:enterpriseSlug/admin/catalog_management"
+              component={routeProps => (
+                <Container className="py-3" fluid>
+                  <MultipleSubscriptionsPage
+                    {...routeProps}
+                    redirectPage="catalog_management"
+                    useCatalog
+                    leadText="Choose a subscription to enroll your learners in courses"
+                    buttonText="Enroll learners"
+                  />
+                </Container>
```

Comment:
nit: It may make sense to abstract this into its own separate, small component (up to you, though; I don't feel strongly about it).